exchanges/request/request_test.go

```diff
@@ -559,77 +513,61 @@ func TestBasicLimiter(t *testing.T) {
 }
 
 func TestEnableDisableRateLimit(t *testing.T) {
-	r, err := New("TestRequest", new(http.Client), WithLimiter(NewBasicRateLimit(time.Minute, 1, 1)))
-	if err != nil {
-		t.Fatal(err)
-	}
-	ctx := context.Background()
+	rl := NewBasicRateLimit(50*time.Millisecond, 1, 1)
+	r, err := New("TestRequest", http.DefaultClient, WithLimiter(rl))
```

Comment:
🦍 Even more pedantic, but still: 
```suggestion
	r, err := New("TestRequest", http.DefaultClient, WithLimiter(NewBasicRateLimit(50*time.Millisecond, 1, 1)))
```